frontend/src/pages/projects/projectSharing/ProjectSharingNameInput.tsx

```diff
@@ -25,6 +25,7 @@ const ProjectSharingNameInput: React.FC<ProjectSharingNameInputProps> = ({
   if (!typeAhead) {
     return (
       <TextInput
+        data-testid={`input ${value}`}
```

Comment:
Test IDs, while not needing to be unique, should be somewhat descriptive. eg. `project-name-input`